### ISSUE TITLE
Confused about PHP7/ZTS

### ISSUE THREAD

**Issue by ClosetGeek-Git** [deuill/go-php#68]
Hello, wondering what the is issue with ZTS in PHP7? For most part handing tsrm resources is easier in php7 (mostly unnecessary in fact).

**Comment by deuill** [comment]
You are correct, things are much better in PHP 7.x than they were in 5.x. I've sort of explained why support was removed [here](https://github.com/deuill/go-php/issues/56#issuecomment-406872919), but it mainly comes down to not really having much time to work on this library. Support across 5.x and 7.x versions makes things a bit more complicated, as well as slight differences between 7.x versions, which are currently unaccounted for.

That is to say, it's (probably) not a large amount of work -- I just haven't done it. I would welcome contributions, however.
